components/SurveyModal.js

```diff
@@ -0,0 +1,566 @@
+      </div>
+      {stepError !== '' && <p className="tji-modal__form__error">{stepError}</p>}
+    </form>
+  );
+}
+
+Step2.propTypes = {
+  validateStep: PropTypes.func.isRequired,
+  updateForm: PropTypes.func.isRequired,
+  stepError: PropTypes.string,
+};
+
+function Step3(props) {
+  const { validateStep, updateForm, stepError } = props;
+  const requiredFields = ['dataSought', 'dataFound'];
+
+  return (
+    <form className="tji-modal__form">
+      <h2 className="tji-modal__title">I'm looking for data on...</h2>
+      <p className="tji-modal__description">
+        To ensure we're collecting data that is useful to you, we need feedback on the type of data you're searching
+        for. Use the input below to describe the type of data you're looking for.
+      </p>
+      <fieldset>
+        <input
+          type="text"
+          name="data-sought"
+          placeholder="ex: Officers shot"
+          onChange={event => updateForm(event, 'dataSought')}
+        />
+      </fieldset>
+      <fieldset>
+        <p>I found the data I was looking for...</p>
+        <div className="tji-modal__fieldset-flex">
+          <div className="tji-modal__form-col-2 tji-modal__form-radio-group">
+            <label htmlFor="datafound-yes">
+              <input
+                id="datafound-yes"
+                type="radio"
+                name="datafound"
+                value="yes"
+                onChange={event => updateForm(event, 'dataFound')}
+              />
+              Yes!
+            </label>
+          </div>
+          <div className="tji-modal__form-col-2 tji-modal__form-radio-group">
+            <label htmlFor="datafound-no">
+              <input
+                id="datafound-no"
+                type="radio"
+                name="datafound"
+                value="no"
+                onChange={event => updateForm(event, 'dataFound')}
+              />
+              No
+            </label>
+          </div>
+        </div>
+      </fieldset>
+      <div className="tji-modal__actions">
+        <button
+          type="button"
+          onClick={() => {
+            validateStep(requiredFields);
+          }}
+          className="btn btn--primary"
+        >
+          Continue
+        </button>
+      </div>
+      {stepError !== '' && <p className="tji-modal__form__error">{stepError}</p>}
+    </form>
+  );
+}
+
+Step3.propTypes = {
+  validateStep: PropTypes.func.isRequired,
+  updateForm: PropTypes.func.isRequired,
+  stepError: PropTypes.string,
+};
+
+function Step4(props) {
+  const { skipStep, validateStep, updateForm, stepError } = props;
+  const requiredFields = ['firstName', 'lastName', 'email'];
+  return (
+    <form
+      id="mailchimp-form"
+      name="mailchimp-form"
+      className="tji-modal__form"
+      // action="https://texasjusticeinitiative.us18.list-manage.com/subscribe/post?u=fd262cb4a5fc0bafb38da2e22&amp;id=2663621fac"
+      // method="post"
+      // target="_blank"
```

Comment:
Think we should be able to dump these commented lines.

Reply:
Get out of here comments!